v23.1/fips-compliance.md

```diff
@@ -0,0 +1,355 @@
+~~~
+
+~~~ shell
+go1.19.5fips
+~~~
+
+This indicates that CockroachDB was built using [Red Hat's Go FIPS with OpenSSL toolchain](https://github.com/golang-fips/go).
+
+## Use the FIPS-ready CockroachDB Docker image
+
+1. If necessary, enable FIPS mode on the Docker host. The FIPS-ready CockroachDB Docker image must run on a Docker host with FIPS mode enabled. To enable FIPS mode in the Docker host kernel, refer to [Enable FIPS mode](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening) in Red Hat's documentation. To verify that FIPS mode is enabled, refer to [Verify that the kernel enforces FIPS mode](#verify-that-the-kernel-enforces-fips-mode).
+1. Visit [Releases](/docs/releases/index.html?filters=docker) and copy the name of a FIPS-ready Docker image tag.
+1. Pull the Docker image locally, create a new container that uses it, run the container, and attach to it. This example gives the running container the name `cockroachdb-fips-container`. Replace `<image_tag>` with the Docker image tag you copied.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker run <image_tag> --name="cockroachdb-fips-container" -i
+    ~~~
+
+1. In the Docker host, [verify that the kernel enforces FIPS mode](#verify-that-the-kernel-enforces-fips-mode).
+
+    {{site.data.alerts.callout_info}}
+    Do not attempt to run the `sysctl` test from within the running container.
+    {{site.data.alerts.end}}
+
+1. In the running container, [verify that CockroachDB is FIPS-ready](#verify-that-cockroachdb-is-fips-ready).
+1. To stop the container, use `CTRL-C`. To detach from the container but keep it running in the background, use the sequence `CTRL+P+CTRL+Q`.
+
+## Details about cryptographic algorithms
+
+This section provides more information about the cryptographic algorithms and key lengths used by FIPS-ready CockroachDB.
+
+### Authentication
+
+#### Inter-node and node identity
+
+**Algorithm**: [TLS 1.3 (RFC 8446)](https://www.rfc-editor.org/rfc/rfc8446)
+
+Refer to [Using Digital Certificates with CockroachDB](authentication.html#using-digital-certificates-with-cockroachdb).
+
+#### Client identity
+
+##### Password authentication
+
+**Algorithm**: `bcrypt` or `scram-sha-256`.
+
+Refer to [SASL/SCRAM-SHA-256 Secure Password-based Authentication](security-reference/scram-authentication.html).
+
+##### Client certificates
+
+**Algorithm**: [TLS 1.3 (RFC 8446)](https://www.rfc-editor.org/rfc/rfc8446)
+
+Refer to [Using Digital Certificates with CockroachDB](authentication.html#using-digital-certificates-with-cockroachdb).
+
+##### GSSAPI / Kerberos
+
+Not supported for FIPS-ready deployments.
+
+##### SASL / SCRAM password authentication
+
+**Algorithm**: `scram-sha-256`
+
+Refer to [SASL/SCRAM-SHA-256 Secure Password-based Authentication](security-reference/scram-authentication.html).
+
+##### JSON Web Tokens (JWTs)
```

Comment:
```suggestion
##### JSON web tokens (JWTs)
```

Reply:
I don't think this is correct, the name of the thing is JSON Web Tokens.